content/docs/user-guide/experiment-management/comparing-experiments.md

```diff
@@ -224,264 +144,151 @@ $ dvc exp show --csv | csvstat
-          }
-        },
-        "queued": false,
-        "running": false,
-        "executor": null,
-        "metrics": {
-          "metrics.json": {
-            "data": {
-              "loss": 0.236574187874794,
-              "acc": 0.9126999974250793
-            }
-          }
-        }
-      }
-    }
+[
+  {
+    "Experiment": "paled-acre",
+    "rev": "883442c",
+    "Created": "Apr 19, 2023",
+    "dice_multi": 0.8590125166103912,
+    "train.arch": "squeezenet1_1"
+  },
+  {
+    "Experiment": "vocal-suer",
+    "rev": "231e504",
+    "Created": "Apr 19, 2023",
+    "dice_multi": 0.8997336177828745,
+    "train.arch": "resnet34"
   },
-  "23ceb4a6623d46a5c906f265b5846ef2f332f756": {
-    "baseline": {...}
-    "ca55d7d4763d74bfeac2ca08369489c2881c03ac": {...}
-    "6d13f334bbdd6ea40412d4ce9b5d81d6abb731ee": {...}
-    "69503c61df98752b07772415e0715655dc4ccaa0": {...}
-    "49779aa9c863a5503008eae08b03a6e707f8ddad": {...}
+  {
+    "Experiment": "banal-hogs",
+    "rev": "ff4a08a",
+    "Created": "Apr 19, 2023",
+    "dice_multi": 0.8758231459806097,
+    "train.arch": "alexnet"
   }
-}
+]
 ```
 
-The result is a JSON dictionary with the keys `workspace` and the Git commit IDs
-in SHA-256 digest. These commit IDs are those that the experiments are
-originated from. These commits contain a `baseline` key that denotes the
-baseline experiment of that particular commit and experiment IDs in SHA-256
-digest. The example output above is shortened to show this structure.
+The format returned by `dvc.api.exp_show()` can be directly converted to a
+[Pandas DataFrame](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.html):
 
-Each experiment entry has the following structure:
+```py
+import dvc.api
+import pandas as pd
 
-```json
-"49779aa9c863a5503008eae08b03a6e707f8ddad": {
-      "data": {
-        "timestamp": "2021-09-09T12:53:51",
-        "params": {
-          "params.yaml": {
-            "data": {
-              "train": {
-                "epochs": 10
-              },
-              "model": {
-                "conv_units": 96
-              }
-            }
-          }
-        },
-        "queued": false,
-        "running": false,
-        "executor": null,
-        "metrics": {
-          "metrics.json": {
-            "data": {
-              "loss": 0.2309877723455429,
-              "acc": 0.916100025177002
-            }
-          }
-        },
-        "name": "cnn-64"
-      }
-    }
+df = pd.DataFrame(dvc.api.exp_show())
 ```
 
-As an example, let's feed our experiments to JSON-parsing tool
-[jq](https://stedolan.github.io/jq/) and filter through only metrics:
+</details>
 
-```cli
-$ dvc exp show --json | jq '.[].baseline.data.metrics'
-{
-  "metrics.json": {
-    "data": {
-      "loss": 0.236574187874794,
-      "acc": 0.9126999974250793
-    }
-  }
-}
-{
-  "metrics.json": {
-    "data": {
-      "loss": 0.236574187874794,
-      "acc": 0.9126999974250793
-    }
-  }
-}
-```
+## Compare plots
```

Comment:
feels like it should be more prominent tbh, part of the compare functionality ... I wonder if should include some screenshots/previews at least.

Also, is it the right title for end users - "compare plots"? We are still comparing metrics, right? Just in a different format. What is the easiest for people to understand?

Reply:
> feels like it should be more prominent tbh, part of the compare functionality ... I wonder if should include some screenshots/previews at least.

Added a more expansive section. PTAL.

I didn't want to repeat https://dvc.org/doc/user-guide/experiment-management/visualizing-plots#comparing-plots, but I think it's better even if it's slightly repetitive.

> Also, is it the right title for end users - "compare plots"? We are still comparing metrics, right? Just in a different format. What is the easiest for people to understand?

🤔 I think it's easy for people to understand regardless of the title. Plots aren't a novel concept and neither is comparing them as part of experiment tracking, so no strong opinion from me, but I doubt it's too important.

Reply:
> Plots aren't a novel concept and neither is comparing them as part of experiment tracking, so no strong opinion from me, but I doubt it's too important.

Yep, I think my question is more about even - do we need a separate section at all for this? (since it's nothing special - should it be just part of the main section, w/o subtitles, etc).

Reply:
yep, nw, it's not a blocker at all...